main.cpp

```diff
@@ -200,166 +217,152 @@ struct iPhone FIXME: type names need to begin with a capital letter
         void changeISO(float newISOValue);
         void captureImage(int numberOfImages);
     };
-
-//     1) Take photos
     void takePhotos(int numberOfPhotos, std::string typeOfPhoto, Camera theCamera);
-//     2) Make phone calls
     void makePhoneCall(int phoneNumber, std::string nameOfcontact);
-//     3) Go on internet
-    void connectToInternet(int IPAddress);
+    void connectToInternet(int ipAddress);
 };
 
-/*
-Thing 3) My Refrigerator
-    1) Number of eggs (int)
-    2) Number of vegetables (int)
-    3) Current temperature of fridge (float)
-    4) Percentage full (float)
-    5) Current temperature of freezer (float)
-    1) Change the temperature of fridge
-    2) Change the temperature of freezer
-    3) Return number food items in fridge
- */
+void IPhone::takePhotos(int numberOfPhotos, std::string typeOfPhoto, IPhone::Camera theCamera)
+{
+    for(int i = 0; i < numberOfPhotos; i++)
+        {
+            std::cout << typeOfPhoto << " " << theCamera.manufactuer;   
+        }
+}
```

Comment:
loops have not been covered yet.
Please remove this and write something else instead.
- [ ] revise